schema/files/OCFManifestFile.schema.json

```diff
@@ -100,7 +98,14 @@
       "title": "Cap Table - Financing File(s) Array",
       "description": "List of files containing lists of financings, indexed from the file containing the first such object created to the file containing the last (See separate /schema/files/financings_file schema to validate loaded files)",
       "type": "array",
-      "minItems": 1,
+      "items": {
+        "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/types/File.schema.json"
+      }
+    },
+    "documents_files": {
+      "title": "Cap Table - Document File(s) Array",
+      "description": "List of files containing lists of document paths, indexed from the file containing the first such object created to the file containing the last (See separate /schema/files/documents_file schema to validate loaded files)",
+      "type": "array",
       "items": {
         "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/types/File.schema.json"
       }
```

Comment:
Note in the `required` array (below the shown diff) we have the original 7 non-manifest files as required.

When `financings_files` was added, we did not require it. Similar we will not require `documents_files` at this time.

I also noticed some files having `"minItems": 1` but not all. Since adding `"minItems": 1` where it doesn't exist here is technically a breaking change, I removed `"minItems": 1` from the couple places it did exist (empty array is fine in this scenario IMO)